.gitmodules

```diff
@@ -11,3 +11,11 @@
 [submodule "components/mpas-source"]
 	path = components/mpas-source
 	url = https://github.com/MPAS-Dev/MPAS-Model.git
+[submodule "externals/scorpio"]
+	path = externals/scorpio
+  url = git@github.com:E3SM-Project/scorpio.git
```

Comment:
Maint branches use https protocol for submodules.

Reply:
ok, I will change that

Reply:
This is fixed on the branch now